packages/aws-rfdk/lib/core/scripts/powershell/configureCloudWatchAgent.ps1

```diff
@@ -3,68 +3,83 @@
 
 # This script downloads, installs and configures the cloudwatch agent. Must be run as sudo capable user.
 # Arguments:
-# $1: SSM parameter name
-# [Switch] s: Skips the verification of the cloudwatch agent installer
+# $1: The region this script is running in
+# $2: SSM parameter name
+# [Switch] s: Skips the verification of the CloudWatch agent installer
+# [Switch] i: Attempts to install the CloudWatch agent
 
 param (
-    [Parameter(Mandatory=$True)] $ssmParameterName,
+  [Parameter(Mandatory=$True)] $region,
+  [Parameter(Mandatory=$True)] $ssmParameterName,
 
-    # This parameter name intentionally does not follow the standard PascalCase style used in Powershell
-    # so that in our Typescript code, we can pass the same flag regardless of the script being called
-    [switch] $s = $False
+  # These parameter names intentionally do not follow the standard PascalCase style used in Powershell
+  # so that in our Typescript code, we can pass the same flag regardless of the script being called
+  [switch] $i = $True,
```

Comment:
We have the default value for this flag as false in the bash script. Should we change the default value here to false as well?

Reply:
Yes that should be false, I've updated it.